src/SourceBuild/content/repo-projects/installer.proj

```diff
@@ -12,6 +12,9 @@
     <OverrideTargetRid Condition="'$(TargetOS)' == 'Windows_NT'">win-$(Platform)</OverrideTargetRid>
     <OSNameOverride>$(OverrideTargetRid.Substring(0, $(OverrideTargetRid.IndexOf("-"))))</OSNameOverride>
 
+    <_buildVertical Condition="'$(DotNetBuildVertical)' == ''">false</_buildVertical>
+    <_buildVertical Condition="'$(DotNetBuildVertical)' == 'true'">true</_buildVertical>
```

Comment:
In order to avoid repeating this for each repo that requires it, I would suggest defining it in src/SourceBuild/content/Directory.Build.props.

Also, can it be simplified to just the following so you don't need `_buildVertical`:

```xml
<DotNetBuildVertical Condition="'$(DotNetBuildVertical)' == ''">false</DotNetBuildVertical>
```

Reply:
So just using the same property name from the environment and... that's really elegant!